cli/src/funTest/kotlin/MainTest.kt

```diff
@@ -100,6 +100,29 @@ class MainTest : StringSpec() {
             lines.next() shouldBe "\tNPM"
         }
 
+        "Output formats are deduplicated" {
+            val inputDir = File(projectDir, "gradle")
+
+            // Redirect standard output to a stream.
+            val standardOut = System.out
+            val streamOut = ByteArrayOutputStream()
+            System.setOut(PrintStream(streamOut))
+
+            Main.main(arrayOf(
+                    "analyze",
+                    "-m", "Gradle",
+                    "-i", inputDir.path,
+                    "-o", File(outputDir, "gradle").path,
+                    "-f", "json,yaml,json"
+            ))
+
+            // Restore standard output.
+            System.setOut(standardOut)
```

Comment:
Should go to a finally block in case the call above throws an exception.

Reply:
Done (in a new commit, as more tests were affected).